src/OrchardCore.Modules/OrchardCore.ContentFields/Media/MediaShapes.cs

```diff
@@ -1,8 +1,11 @@
 using OrchardCore.DisplayManagement;
 using OrchardCore.DisplayManagement.Descriptors;
+using OrchardCore.Modules;
 
 namespace OrchardCore.ContentFields.Media
 {
+    [Feature("OrchardCore.ContentFields")]
+    [RequireFeatures("OrchardCore.Media")]
```

Comment:
Need to be tried but I think the feature attribute is not needed when related to the module main feature.

But the require feature attribute seems to make sense.

Reply:
done